docs/Makefile

```diff
@@ -9,12 +9,16 @@ SOURCEDIR     = .
 BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
-help:
+help:	fledge_plugins.rst
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 .PHONY: help Makefile
 
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
\ No newline at end of file
+%: Makefile fledge_plugins.rst
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+fledge_plugins.rst: 
+	@echo Building page with table of plugins
+	@bash scripts/fledge_plugin_list fledge_plugins.rst
```

Comment:
Better to add `clean` target - remove fledge_plugins.rst file. As of now it removes only _build content i.e from BUILDDIR
Sphinx inbuilt variable